yt/frontends/boxlib/data_structures.py

```diff
@@ -739,6 +739,10 @@ def _parse_cparams(self):
                 param, vals = (s.strip() for s in line.split("="))
             except ValueError:
                 continue
+            # Castro and Maestro mark overridden defaults with a "[*]" before
+            # the parameter name
+            if param.startswith("[*]"):
+                param = param.removeprefix("[*]").strip()
```

Comment:
`str.removeprefix` returns early if it has nothing to do so we shouldn't need to have a conditional. Maybe I'm missing something ?
```suggestion
            param = param.removeprefix("[*]").strip()
```

Reply:
I noticed this shortly after making the PR, but forgot to push.